From aad67ace8a3f2956d632ef4a83ef7a953817f31d Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Sun, 24 May 2026 21:31:46 +0300
Subject: [PATCH] Fix build issues

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 Android.bp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Android.bp b/Android.bp
index 3631635..be16062 100644
--- a/Android.bp
+++ b/Android.bp
@@ -12,7 +12,6 @@ cc_defaults {
     cflags: [
 	"-g3",
 	"-Werror",
-	"-D_FORTIFY_SOURCE=2",
 	"-DYAML_VERSION_MAJOR=" + YAML_MAJOR,
 	"-DYAML_VERSION_MINOR=" + YAML_MINOR,
 	"-DYAML_VERSION_PATCH=" + YAML_PATCH,
@@ -28,7 +27,7 @@ cc_library_headers {
     ],
 }
 
-cc_library_shared {
+cc_library {
     name: "libyaml",
     vendor: true,
     defaults: [
-- 
2.51.0

